pkg/extensions/search/common/common.go

```diff
@@ -11,11 +11,13 @@ import (
 )
 
 const (
-	AnnotationLabels           = "org.label-schema.labels"
-	LabelAnnotationCreated     = "org.label-schema.build-date"
-	LabelAnnotationVendor      = "org.label-schema.vendor"
-	LabelAnnotationDescription = "org.label-schema.description"
-	LabelAnnotationLicenses    = "org.label-schema.license"
+	AnnotationLabels             = "org.label-schema.labels"
```

Comment:
Why use these? instead of https://github.com/opencontainers/image-spec/blob/main/annotations.md and standardize on them.

Reply:
I'm using manifest annotations, the standard ones, and I'm falling back to these labels (from config blobs) in case annotations are not present on manifests

Reply:
for example:
return GetAnnotationValue(annotations, ispec.AnnotationDocumentation, LabelAnnotationDocumentation)

ispec.AnnotationDocumentation is the standard one
LabelAnnotationDocumentation is the fallback label

Reply:
See https://github.com/opencontainers/image-spec/blob/main/annotations.md#back-compatibility-with-label-schema